### PR TITLE
[Docs] update readme

### PR DESCRIPTION
**Pull request by fangyixiao18**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +77,18 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
 
 Differences between MMSelfSup and OpenSelfSup codebases can be found in [compatibility.md](docs/en/compatibility.md).
 
+### Preview of 1.x version
+
+A brand new version of **MMSelfSup v1.0.0rc1** was released in 01/09/2022:
+
+Highlights of the new version:
```
Comment (YuanLiuuuuuu) [inline README.md:84]: more highlights can be listed here, for example, 
- support MAE with different pre-training epochs and backbones of different scales
- more concise API
- higher accurcy for some algorithms

File: README.md [hunk 2]
```diff
@@ -75,6 +77,18 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
...
+
+- Based on [MMEngine](https://github.com/open-mmlab/mmengine) and [MMCV](https://github.com/open-mmlab/mmcv/tree/2.x).
+- Released with refactor.
+- Refine all [documents](https://mmselfsup.readthedocs.io/en/1.x/).
+
+Find more new features in [1.x branch](https://github.com/open-mmlab/mmselfsup/tree/1.x). Issues and PRs are welcome!
+
 ## Installation
 
 MMSelfSup depends on [PyTorch](https://pytorch.org/), [MMCV](https://github.com/open-mmlab/mmcv) and [MMClassification](https://github.com/open-mmlab/mmclassification).
```
Comment (YuanLiuuuuuu) [inline README.md]:
```suggestion
MMSelfSup relies on [PyTorch](https://pytorch.org/), [MMCV](https://github.com/open-mmlab/mmcv) and [MMClassification](https://github.com/open-mmlab/mmclassification).
```

File: README.md [hunk 3]
```diff
@@ -83,7 +97,7 @@ Please refer to [install.md](docs/en/install.md) for more detailed instruction.
 
 ## Get Started
 
-Please refer to [prepare_data.md](docs/en/prepare_data.md) for dataset preparation and [get_started.md](docs/en/get_started.md) for the basic usage of MMSelfSup.
+Please refer to [prepare_data.md](docs/en/prepare_data.md) for dataset preparation, [get_started.md](docs/en/get_started.md) for the basic usage and [benchmarks.md](docs/en/tutorials/6_benchmarks.md) for running benchmarks.
 
 We also provides tutorials for more details:
```
Comment (YuanLiuuuuuu) [inline README.md]:
```suggestion
We also provides more detailed tutorials:
```

File: README_zh-CN.md [hunk 4]
```diff
@@ -65,6 +65,8 @@ MMSelfSup 是一个基于 PyTorch 实现的开源自监督表征学习工具箱
 
 ## 更新
 
+### 稳定版本
```
Comment (YuanLiuuuuuu) [inline README_zh-CN.md:68]: Above comments are also applied to this Chinese version.

File: README.md [hunk 5]
```diff
@@ -75,25 +77,40 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
...
+### Preview of 1.x version
+
+A brand new version of **MMSelfSup v1.0.0rc1** was released in 01/09/2022:
+
+Highlights of the new version:
+
+- Based on [MMEngine](https://github.com/open-mmlab/mmengine) and [MMCV](https://github.com/open-mmlab/mmcv/tree/2.x).
+- Released with refactor.
+- Refine all [documents](https://mmselfsup.readthedocs.io/en/1.x/).
+- Support MAE with different pre-training epochs and backbones of different scales.
```
Comment (YuanLiuuuuuu) [inline README.md]: SimMIM should also be added

**Review by YuanLiuuuuuu — APPROVED** [review]

**Review by tonysy — APPROVED** [review]

**Pull request merged**